Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -338,6 +338,7 @@ private void ReturnToMainMenu()
         PlayerInputManagerController.Singleton.ChangeInputMaps("Menu");
 
         // Mirror pulls us to the main menu automatically
-        NetworkManager.singleton.StopHost();
+        if (isServer)
+            NetworkManager.singleton.ServerChangeScene(Scenes.Menu);
```

Comment:
should turn this into server changing scene to menu and clients changing to client lobby

Reply:
Take a look at this (do non additive):
`conn.Send(new SceneMessage() { sceneName = msg.sceneName, sceneOperation = SceneOperation.LoadAdditive });`